### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by damacus**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/damacus/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
```
Comment (damacus) [inline ReadMe.md]: !fix

Comment (damacus) [inline ReadMe.md]: comment

Comment (damacus) [inline ReadMe.md]: !fixed

**Pull request merged**